inspector/src/modules/log_module.ts

```diff
@@ -783,18 +867,33 @@ function setEnabledFilterButtonStyle(
  * @param {string} logTxt
  * @returns {HTMLElement}
  */
-export function createLogElement(logTxt: string): HTMLElement {
+export function createLogElement(
+  logTxt: string,
+  logView: ObservableState<LogViewState>,
+  configState: ObservableState<ConfigState>,
+  ): HTMLElement {
   let namespace;
-  let formattedMsg = logTxt;
-  const indexOfNamespaceStart = logTxt.indexOf("[");
+  // let formattedMsg = logTxt;
+  const timeRegex = /^(\d+(?:.)?(?:\d+)?) (.*)$/
```

Comment:
I think that browsers are able to optimize regex compilation when the same is used in top scope.